File: app/src/main/java/org/evcs/android/model/Address.java
package org.evcs.android.model;

import java.io.Serializable;

public class Address implements Serializable {
    String streetAddress;
    String city;
    String state;
    String zipCode;
    String country;

    @Override
    public String toString() {
        return streetAddress + ", " + city + ", " + state + " " + zipCode;
    }
}

File: app/src/main/java/org/evcs/android/model/Address.kt
package org.evcs.android.model

import java.io.Serializable

class Address : Serializable {

    var streetAddress: String? = null
    var city: String? = null
    var state: String? = null
    var zipCode: String? = null
    var country: String? = null

    override fun toString(): String {
        return "$streetAddress,\n$city, $state $zipCode"
    }
}